### PR TITLE
Uueoa-Esa, Part 4: Untouched Lands

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds several ruins for the Untouched Lands of Moghes

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

**Comment by RustingWithYou** [comment]
!review

File: maps/random_ruins/exoplanets/moghes/moghes_bar.dm [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+	r_pocket = /obj/item/storage/wallet/random
+	l_ear = null
+	id = /obj/item/card/id
+
+/obj/outfit/admin/moghes_bar_inkeeper/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/uniform_color = "[pick("#473b13", "#134718", "#0a6959")]"
+	if(H?.w_uniform)
+		H.w_uniform.color = uniform_color
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```
Comment (FluffyGhoster) [inline maps/random_ruins/exoplanets/moghes/moghes_bar.dm:57]: Might aswell return early if no human is passed in as parameter, also useless var

File: maps/random_ruins/exoplanets/moghes/moghes_bar.dm [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+	id = /obj/item/card/id
+
+/obj/outfit/admin/moghes_bar_server/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/uniform_color = "[pick("#473b13", "#134718", "#0a6959")]"
+	if(H?.w_uniform)
+		H.w_uniform.color = uniform_color
+	if(H?.wear_suit)
+		H.wear_suit.color = "#423509"
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```
Comment (FluffyGhoster) [inline maps/random_ruins/exoplanets/moghes/moghes_bar.dm:98]: As above

File: maps/random_ruins/exoplanets/moghes/moghes_bar.dm [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+	id = null
+
+/obj/outfit/admin/moghes_bar_patron/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	var/uniform_color = "[pick("#473b13", "#134718", "#0a6959", "#d2d9d8", "#262928")]"
+	if(H?.w_uniform)
+		H.w_uniform.color = uniform_color
+	if(H?.wear_suit)
+		H.wear_suit.color = "#423509"
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```
Comment (FluffyGhoster) [inline maps/random_ruins/exoplanets/moghes/moghes_bar.dm:152]: As above

File: maps/random_ruins/exoplanets/moghes/moghes_diona_traders.dm [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+		/obj/item/device/uv_light = 1
+	)
+	r_pocket = /obj/item/storage/wallet
+
+/obj/outfit/admin/moghes_diona/assistant/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(H?.w_uniform)
+		H.w_uniform.color = "#EAEAEA"
+		H.w_uniform.accent_color = "#EAEAEA"
+	if(H?.wear_suit)
+		H.wear_suit.color = "#42330f"
```
Comment (FluffyGhoster) [inline maps/random_ruins/exoplanets/moghes/moghes_diona_traders.dm:106]: As above minus the var

File: maps/random_ruins/exoplanets/moghes/moghes_heph_mining.dm [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+	if(H?.wear_mask && H.species.has_organ[BP_PHORON_RESERVE])
+		var/obj/item/organ/internal/vaurca/preserve/preserve = H.internal_organs_by_name[BP_PHORON_RESERVE]
+		H.internal = preserve
+		H.internals.icon_state = "internal1"
+	if(H?.wear_suit)
+		H.wear_suit.color = "#2a2b2e"
+	var/obj/item/organ/B = new /obj/item/organ/internal/augment/tool/drill(H)
+	var/obj/item/organ/external/affectedB = H.get_organ(B.parent_organ)
+	B.replaced(H, affectedB)
+	H.update_body()
```
Comment (FluffyGhoster) [inline maps/random_ruins/exoplanets/moghes/moghes_heph_mining.dm:88]: Return early if the parameter is not specified

File: maps/random_ruins/exoplanets/moghes/moghes_thakh.dm [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+		/obj/item/clothing/accessory/poncho/unathimantle/mountain
+	)
+	l_ear = null
+	id = null
+
+/obj/outfit/admin/moghes_thakh/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	if(H?.wear_suit)
+		H.wear_suit.color = "#423509"
+	if(H?.shoes)
+		H.shoes.color = "#423509"
```
Comment (FluffyGhoster) [inline maps/random_ruins/exoplanets/moghes/moghes_thakh.dm:94]: Etc. etc. as above

**Review by Eyeveri — CHANGES_REQUESTED** [review]
Is this intentional? The left and right pews being inverted.
image

All maps look great! Can't find any other reasons to not approve aside from the pews. 

The code looks fine to me on all of this, but again, I'm still learning the code side of this so Dreamy or Fluffy should take another look at them.

**Comment by RustingWithYou** [comment]
> Is this intentional? The left and right pews being inverted. image
> 
> All maps look great! Can't find any other reasons to not approve aside from the pews.
> 
> The code looks fine to me on all of this, but again, I'm still learning the code side of this so Dreamy or Fluffy should take another look at them.

Addressed and replaced with actual pews

**Review by Eyeveri — APPROVED** [review]

**Pull request merged**